src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -1579,7 +1579,10 @@ protected override void DecodeWellKnownAttributeImpl(ref DecodeWellKnownAttribut
                 }
                 else
                 {
-                    diagnostics.Add(ErrorCode.ERR_UnscopedRefAttributeUnsupportedMemberTarget, arguments.AttributeSyntaxOpt.Location);
+                    var code = !this.ContainingModule.UseUpdatedEscapeRules
+                        ? ErrorCode.WRN_UnscopedRefAttributeOldRules
```

Comment:
>WRN_UnscopedRefAttributeOldRules [](http://example.com/codeflow?start=36&length=32)

Are we reducing an existing error to a warning here, in cases where the previous error applied regardless of which rules were used? For instance, when compiling the following with C# 10 and .NET 6:
```csharp
struct S
{
    [UnscopedRef] object P { get; init; }
}
```